.github/workflows/dependency-submission.yml

```
@@ -17,17 +17,20 @@ on:
 
 permissions:
   contents: write
+  dependency-graph: write
```

actionlint (1.7.12)

[error] unknown permission scope "dependency-graph". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)